code/obj/flock/structure/annihilator.dm

```diff
@@ -0,0 +1,144 @@
+/obj/flock_structure/annihilator
+	icon_state = "annihilator-off" // placeholder sprites
+	name = "gnesis fountain"
+	desc = "Some sort of gnesis fountain. The gnesis appears very active."
+	flock_desc = "A defense turret that fires high speed gnesis bolts at nearby projectiles, annihilating them."
+	flock_id = "Annihilator"
```

Comment:
Prefer this changed to something per other comments.

Reply:
Renamed if sounds good